composer.json

```diff
@@ -5,19 +5,19 @@
     "license": "MIT",
     "minimum-stability": "dev",
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.3",
         "ext-json": "*",
-        "symfony/config": "^5.4 | ^6.0 | ^7.0",
-        "symfony/dependency-injection": "^5.4 | ^6.0 | ^7.0",
-        "symfony/http-kernel": "^5.4 | ^6.0 | ^7.0",
-        "symfony/routing": "^5.4 | ^6.0 | ^7.0",
+        "symfony/config": "^6.4.5 | ^7.0",
+        "symfony/dependency-injection": "^6.4.5 | ^7.0",
+        "symfony/http-kernel": "^6.4.5 | ^7.0",
+        "symfony/routing": "^6.4.5 | ^7.0",
         "symfony/deprecation-contracts": "^2.2 | ^3.0"
     },
     "require-dev": {
         "doctrine/orm": "^2.7",
         "doctrine/persistence": "^2.0",
-        "symfony/framework-bundle": "^5.4 | ^6.0 | ^7.0",
-        "symfony/phpunit-bridge": "^5.4 | ^6.0 | ^7.0"
+        "symfony/framework-bundle": "^6.4.5 | ^7.0",
+        "symfony/phpunit-bridge": "^6.4.5 | ^7.0.5"
```

Comment:
PHPUnit Bridge 7.0.0 is unusable with recent 9.x versions of PHPUnit. PHPUnit ships with a `composer.lock` file. 

Note to future self - when you can't figure out why `simple-phpunit` fails to install phpunit -> it's because it's doing a `composer install` instead of a `composer upgrade`.  https://github.com/symfony/symfony/pull/52846